parent-poms/pom.xml

```diff
@@ -943,13 +967,13 @@
 				<artifactId>license-maven-plugin</artifactId>
 				<version>${license.version}</version>
 				<executions>
-					<execution>
-						<id>download-licenses</id>
-						<goals>
-							<goal>download-licenses</goal>
-							<goal>aggregate-add-third-party</goal>
-						</goals>
-						</execution>
+					<!--<execution>-->
```

Comment:
This resulted in strange Forking behavior at runtime within de-etl-framework-java. It was looking for JARs for sibling modules and failing.